dffml/service/dev.py

```diff
@@ -60,57 +61,50 @@
 ]
 
 
-def create_from_skel(name):
+def create_from_skel(self_name):
```

Comment:
This should probably be something more descriptive (`plugin_type` chosen, since we're creating a package of `plugin_type` from skel/)

```suggestion
def create_from_skel(plugin_type):
```

Reply:
Oh, okay!